src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no,   initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.  0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,900&display=swap" rel="stylesheet"
+    >
+
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
```

Comment:
```suggestion
    <div class="card" data-qa="card">
```
It is better to call the block "card" as described in the task description
![image](https://user-images.githubusercontent.com/96409563/217772628-615519dc-8ac7-4113-b398-a9e189735dfa.png)